### PR TITLE
Patient portal sign in 

### LINKED ISSUE #22: Setup Asgardeo Auth for patient portal
**Is your feature request related to a problem? Please describe.**

Setup Asgardeo Auth for the patient portal. Doc: https://wso2.com/asgardeo/

**Describe the solution you'd like**

https://wso2.com/asgardeo/

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by janithRS**
resolves: https://github.com/lk-gov-health-hiu/project-nhdx/issues/22

Screenshot 2023-12-23 at 10 49 14 AM

### CODE REVIEW

File: nehr-patient-portal/app/api/auth/[...nextauth]/route.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export const authOptions: NextAuthOptions = {
+  providers: [
+    {
+      id: "asgardeo",
+      name: "Asgardeo",
+      clientId: process.env.ASGARDEO_CLIENT_ID,
+      clientSecret: process.env.ASGARDEO_CLIENT_SECRET,
+      issuer: process.env.ASGARDEO_ISSUER,
+      userinfo: `https://api.asgardeo.io/t/${process.env.ASGARDEO_ORGANIZATION_NAME}/oauth2/userinfo`,
```
Comment (sameeragunarathne) [inline nehr-patient-portal/app/api/auth/[...nextauth]/route.ts:11]: Shall we use the discovery endpoint itself to get these?

File: nehr-patient-portal/app/i18n.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    ns: namespaces,
+    preload: resources ? [] : i18nConfig.locales
+  });
+
+  return {
+    i18n: i18nInstance,
+    resources: i18nInstance.services.resourceStore.data,
+    t: i18nInstance.t
+  };
+}
```
Comment (sameeragunarathne) [inline nehr-patient-portal/app/i18n.js]: Fix EoFs for all the new files added

Comment (janithRS) [reply nehr-patient-portal/app/i18n.js]: Updated

**Review by sameeragunarathne — APPROVED** [review]

File: nehr-patient-portal/app/api/auth/[...nextauth]/route.ts (hunk 1 above)
Comment (janithRS) [reply nehr-patient-portal/app/api/auth/[...nextauth]/route.ts:11]: We will be looking at this in the future.

Comment (sameeragunarathne) [reply nehr-patient-portal/app/api/auth/[...nextauth]/route.ts:11]: @janithRS can you create a issue and link it here?

**Pull request merged**